Documentation/Blazorise.Docs/Pages/Docs/Extensions/LoadingIndicator/Examples/LoadingIndicatorStructureExample.snippet

```diff
@@ -0,0 +1,16 @@
+﻿@if (Loaded)
+{
+    <div Wrapper>
```

Comment:
It's somewhat difficult to read this structure that is written as `<div Wrapper>`. I must confess I was confused when I first saw it 😅

Reply:
Yes, agreed, but I'm not sure what's a better way. maybe say <wrapper-div> instead?

Reply:
Just having a simplest real example is enough.